<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetResolutionDialog</class>
 <widget class="QDialog" name="GetResolutionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>alpha wolf squadron Sprite Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="widthInputBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#widthInputBox {
background-color: rgba(66, 66, 66, 16);
border-radius: 4px;
color: rgb(180,180,180);
margin: 2px;
}

#widthInputBox:hover {
background-color: rgba(56, 56, 56, 16);
border-radius: 4px;
color: rgb(180,180,180);
margin: 0px;
}
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>width</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightInputBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#heightInputBox {
background-color: rgba(66, 66, 66, 16);
border-radius: 4px;
color: rgb(180,180,180);
margin: 2px;
}

#heightInputBox:hover {
background-color: rgba(56, 56, 56, 16);
border-radius: 4px;
color: rgb(180,180,180);
margin: 0px;
}
</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>height</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#okButton {
border-image: url(:Double Right Filled-100.png);
background: transparent;
background-repeat: none;
margin: 6px;
}
#okButton:hover
{
border-image: url(:Double Right Filled-100-2.png) -5 -5 -5 -5;
border-top: -5px transparent;
border-bottom: -5px transparent;
border-right: -5px transparent;
border-left: -5px transparent;
margin: 2px;
}
#okButton:pressed
{
border-image: url(:Double Right Filled-100-4.png);
margin: 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-radius: 4px;
color: rgb(180,180,180, 180);</string>
   </property>
   <property name="text">
    <string>BACKGROUND</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="transparentBackgroundButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#transparentBackgroundButton {
background-color: rgb(29,29,29);
background-image: url(:transparent pattern 3.png);
border: 0px transparent;
border-radius: 6px;
margin: 7px;
}

#transparentBackgroundButton:hover {
background-color: rgb(29,29,29);
background-image: url(:transparent pattern 3.png);
border: 0px transparent;
border-radius: 6px;
margin: 0px;
}

#transparentBackgroundButton:checked {
background-color: rgb(29,29,29, 230);
background-image: url(:transparent pattern 3.png);
border: 2px solid rgb(232,0,71,230);
border-radius: 6px;
margin: 0px;
}

#transparentBackgroundButton:unchecked {
background-color: rgb(29,29,29);
background-image: url(:transparent pattern 3.png);
border: 0px transparent;
border-radius: 6px;
margin: 7px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="whiteBackgroundButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#whiteBackgroundButton {
background-color: rgb(253,253,253);
border: 0px transparent;
border-radius: 6px;
margin: 7px;
}

#whiteBackgroundButton:hover {
background-color: rgb(255,255,255);
border: 0px transparent;
border-radius: 6px;
margin: 0px;
}

#whiteBackgroundButton:checked {
background-color: rgb(253,253,253);
border: 2px solid rgb(232,0,71,230);
border-radius: 6px;
margin: 0px;
}

#whiteBackgroundButton:unchecked {
background-color: rgb(253,253,253);
border: 0px transparent;
border-radius: 6px;
margin: 7px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="blackBackgroundButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#blackBackgroundButton {
background-color: rgb(14,14,18);
border: 0px transparent;
border-radius: 6px;
margin: 7px;
}

#blackBackgroundButton:hover {
background-color: rgb(14,14,18);
border: 0px transparent;
border-radius: 6px;
margin: 0px;
}

#blackBackgroundButton:checked {
background-color: rgb(14,14,18);
border: 0px transparent;
border: 2px solid rgb(232,0,71,230);
margin: 0px;
}

#blackBackgroundButton:unchecked {
background-color: rgb(14,14,18);
border: 0px transparent;
border-radius: 6px;
margin: 7px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>30</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-radius: 4px;
color: rgb(180,180,180, 180);</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>30</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-radius: 4px;
color: rgb(180,180,180, 180);</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>PX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>320</width>
     <height>175</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:credits.png);
color: rgb(46,46,48);
margin: -1px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>299</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(66, 66, 66, 16);
border-radius: 10px;
color: rgb(180,180,180);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>widthInputBox</zorder>
  <zorder>heightInputBox</zorder>
  <zorder>okButton</zorder>
  <zorder>label_6</zorder>
  <zorder>transparentBackgroundButton</zorder>
  <zorder>whiteBackgroundButton</zorder>
  <zorder>blackBackgroundButton</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
